Gemfile

```diff
@@ -4,6 +4,7 @@ ruby "2.3.0"
 
 gem "autoprefixer-rails"
 gem "bourbon", "5.0.0.beta.3"
+gem "clearance"
```

Comment:
You might want to lock to the 1.x series before the 2.x series gets released some time this year.